### PR TITLE
chore(logging): Migrate some logs to ILogger

### PR DESCRIPTION
**Pull request by ppardalj**
This pull requests replaces some common usages of `Console.WriteLine` for calls to `ILogger`.

### CODE REVIEW

**Review by kamronbatman — APPROVED** [review]

**Comment by kamronbatman** [comment]
Looks great, thank you!
Is this PR ready to merge, or should I wait for more commits to come in?

File: Projects/UOContent/Accounting/AccountHandler.cs [hunk 1]
```diff
@@ -319,7 +323,7 @@ private static Account CreateAccount(NetState state, string un, string pw)
                 return null;
             }
 
-            Console.WriteLine("Login: {0}: Creating new account '{1}'", state, un);
+            logger.Information("Login: {0}: Creating new account '{1}'", state, un);
```
Comment (karolswdev) [inline Projects/UOContent/Accounting/AccountHandler.cs:326]: This is an incorrect usage of Serilog and the Key Value Pair paradigm. Rename the parameters from {0}, {1}, {n} to actual parameters such as ``AccountName``, ``NumberOfAccounts```, etc.

**Review by karolswdev — CHANGES_REQUESTED** [review]
Please revisit my comment and apply it to all affected portions of this PR.

**Comment by karolswdev** [comment]
> Looks great, thank you!
> Is this PR ready to merge, or should I wait for more commits to come in?
> 
> Note: I have a PR that this is stepping over. I'll remove my changes that clobber in favor of this PR.

I would urge you to revisit this PR and not merge it in its current state. You should not use {0}, {1}, {2} as parameters for the logger. You should use meaningful variable names so that when you use a structured logger (such as Seq) you can do actual queries and parameter matching.

**Comment by kamronbatman** [comment]
> > Looks great, thank you!
> > Is this PR ready to merge, or should I wait for more commits to come in?
> > Note: I have a PR that this is stepping over. I'll remove my changes that clobber in favor of this PR.
> 
> I would urge you to revisit this PR and not merge it in its current state. You should not use {0}, {1}, {2} as parameters for the logger. You should use meaningful variable names so that when you use a structured logger (such as Seq) you can do actual queries and parameter matching.

yeah I know, I was simply going to clean that up later. For now the logging is just mimicking console writelines. The actual structured logging will have to come later I think since it is quite a bit more work.

Thoughts on this?

File: Projects/UOContent/Accounting/AccountHandler.cs (hunk 1 above)
Comment (kamronbatman) [reply Projects/UOContent/Accounting/AccountHandler.cs:326]: Yeah I was thinking about whether to flag this or just fix it myself in a subsequent PR. Are you thinking about this:
```cs
logger.Information("Login: {state}: Creating new account '{un}'");
```

Or something else? I am new to Serilog, so a concrete example would be great. Thanks!

Comment (immaciek) [reply Projects/UOContent/Accounting/AccountHandler.cs:326]: `logger.Information("Login: {State}: Creating new account {Username}", state, un)`

https://github.com/serilog/serilog/wiki/Structured-Data

Note we are only using a hardcoded console sink at the moment so it won't have any effect

Comment (kamronbatman) [reply Projects/UOContent/Accounting/AccountHandler.cs:326]: Network, commands, player speech, etc can be handled by different loggers, of which we can sink to files appropriately.
Let's handle that in another iteration.

Comment (ppardalj) [reply Projects/UOContent/Accounting/AccountHandler.cs:326]: @karolswdev I'm aware of that, but as @immaciek mentioned, I'm trying to tackle the problem iteratively. I can handle the parameter names in another iteration.

**Comment by ppardalj** [comment]
> > > Looks great, thank you!
> > > Is this PR ready to merge, or should I wait for more commits to come in?
> > > Note: I have a PR that this is stepping over. I'll remove my changes that clobber in favor of this PR.
> > 
> > 
> > I would urge you to revisit this PR and not merge it in its current state. You should not use {0}, {1}, {2} as parameters for the logger. You should use meaningful variable names so that when you use a structured logger (such as Seq) you can do actual queries and parameter matching.
> 
> yeah I know, I was simply going to clean that up later. For now the logging is just mimicking console writelines. The actual structured logging will have to come later I think since it is quite a bit more work.
> 
> Thoughts on this?

Exactly this is the point. I would merge this PR asap as is to move on and not block other PR's so that other people can start working on improving if they feel like to. IMHO whatever small step towards the right direction is ok.

Feel free to merge this as is from my side, if you think there's some adjustment needed let me know and I'll change it asap :)

Thanks everyone!!

**Pull request merged**